kostyor/rest_api.py

```diff
@@ -132,7 +132,7 @@ def discover_cluster():
         for host, service in six.iteritems(host_service_map):
             for s in service:
                 db_api.create_service(s, hosts_ids[host],
-                                      constants.NOT_READY_FOR_UPGRADE)
+                                      constants.UNDEFINED)
```

Comment:
But method create_service expects service version, which is undefined(or unknown) - https://github.com/sc68cal/Kostyor/blob/master/kostyor/db/api.py#L101